Framework/Algorithms/src/PolarizationCorrections/HeliumAnalyserEfficiency.cpp

```diff
@@ -69,6 +71,16 @@ void HeliumAnalyserEfficiency::init() {
                   "Whether the algorithm should ignore a poor chi-squared (fit cost value) of greater than 1 and "
                   "therefore not throw an error",
                   Direction::Input);
+  declareProperty(std::make_unique<WorkspaceProperty<>>(PropertyNames::OUTPUT_WORKSPACE, "", Direction::Output),
+                  "Helium analyzer efficiency as a function of wavelength");
+  declareProperty(std::make_unique<WorkspaceProperty<MatrixWorkspace>>(std::string(PropertyNames::OUTPUT_FIT_CURVES),
+                                                                       "", Kernel::Direction::Output,
+                                                                       PropertyMode::Optional),
+                  "The name of the matrix workspace containing the calculated fit curve.");
```

Comment:
This sounds like there will just be one curve in the workspace, where actually there is the fit curve, the data, and the difference. Sorry for being pedantic.